plugins/rbac-backend/src/service/policies-rest-api.test.ts

```diff
@@ -321,15 +327,17 @@ describe('REST policies api', () => {
     const router = await server.serve();
     app = express().use(router);
     app.use(errorHandler());
-    jest.clearAllMocks();
-  });
+    return server;
+  }
 
-  it('should build', () => {
+  it('should build', async () => {
+    await createServer();
     expect(app).toBeTruthy();
   });
 
   describe('GET /', () => {
     it('should return a status of Authorized', async () => {
+      await createServer();
```

Comment:
I see that these lines are added to a lot of test cases. Is it possible to add this to the `beforeEach()` to prevent the addition of all of these lines?